cpu/kinetis_common/include/wdog.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2015 PHYTEC Messtechnik GmbH
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    cpu_kinetis_common_wdog Kinetis WDOG
+ * @ingroup     cpu_kinetis_common
+ * @brief       Driver Freescale WDOG Modules.
+ *
+ *              ### WDOG Configuration Example (for periph_conf.h) ###
+ *
+ *
+ *              Optional settings:
+ *
+ * @{
+
+ * @file
+ * @brief       Interface definition for the Kinetis WDOG driver.
+ *
+ * @author      Johann Fischer <j.fischer@phytec.de>
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ */
+
+#ifndef __WDOG_H
+#define __WDOG_H
```

Comment:
Sorry if the comment sounded harsh, I only meant to be direct and to the point..

Like I wrote in another PR (https://github.com/RIOT-OS/RIOT/pull/2355#discussion_r23561946)

>Leading underscores are not really recommended in include guard macro names.
>
>http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf
>
>>    7.1.3 Reserved identifiers
>>    All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

Can you change the include guards to remove the leading double underscores?